### DISCUSSION TITLE
CasaOS 0.4.7 ADguard Home not working ! Solution here !

### DISCUSSION THREAD

**Discussion by cliffliao** [IceWhaleTech/CasaOS#1732] [General]
ADguard Home not working after install. Here is the solution !
After install then finish the configuration but can't login into ADguard Home Web UI.
Click the docker setting of ADguard Home (Three dots of the ICON).

In the port setcion:
Modify 531 to 53 (both TCP and UDP)
Add 3000 mapping to 80 (TCP)

Done !

![ADGuard Home Docker Setting](https://github.com/IceWhaleTech/CasaOS/assets/126017971/c3445ee7-a4bc-41b8-9f65-6c3a70bbe59c)

**Comment by aptonline** [comment]
I tried the same but it reports that port 53 is in use and won't apply the changes.  I have no other 'app' running that could use port 53.  Is the enabled by default within CasaOS?

**Reply by HeavyLvy** [reply]
ik this is old but systemd-resolve is most likely taking it. You'll  need to disable and stop the service, yo can find a tutorial sum where on the internet
